setup.py

```diff
@@ -45,5 +45,9 @@ def readme():
                 'plotly/matplotlylib/mplexporter',
                 'plotly/matplotlylib/mplexporter/renderers'],
       package_data={'plotly': ['package_data/*']},
-      install_requires=['decorator', 'requests', 'six', 'pytz'],
+      install_requires=['decorator',
+                        'nbformat>=4.3',
```

Comment:
Oh ok, yeah, 4.2 should be good. I see `mime-type` listed here: https://github.com/jupyter/nbformat/blob/master/docs/changelog.rst. Earlier versions might be OK too